src/main/resources/application.properties

```diff
@@ -4,6 +4,9 @@ org.eclipsefoundation.adopters.api.ProjectsAPI/mp-rest/url=https://projects.ecli
 quarkus.http.port=8080
 quarkus.http.root-path=/adopters
 
+## CORS settings
+quarkus.http.cors.exposed-headers=*
```

Comment:
My understanding is that this is the default: https://quarkus.io/guides/http-reference